### PR TITLE
Add a Updating the TLS certificate section

### PR DESCRIPTION
**Pull request by gotsysdba**

### CODE REVIEW

**Review by markxnelson — APPROVED** [review]
please get review from @rhondaday too

**Comment by gotsysdba** [comment]
@rhondaday, can you give this change a once-over and approve if good?

File: docs-source/mbaas/content/setup/_index.md [hunk 1]
```diff
@@ -64,25 +64,47 @@ To start the installation, take the following steps:
...
-dashboard_uri = "http://1.2.3.4"
-dashboard_user = "ADMIN"
-kubeconfig_cmd = "oci ce cluster create-kubeconfig --cluster-id ocid1.cluster.oc1.iad.xxx
- --file $HOME/.kube/config --region us-ashburn-1 --token-version 2.0.0 --kube-endpoint PUBLIC_ENDPOINT"
-parse_endpoint = "1.2.3.4/parse"
-```
+
+## TLS
+
+The Oracle Backend for Parse Platform is deployed with a sample "Self-Signed" certificate for TLS.  This results in an "Accept Risk" message when accessing the Parse Dashboard and the sample TLS certificate should not be used for Production deployments.
```
Comment (rhondaday) [inline docs-source/mbaas/content/setup/_index.md:90]: Change "Self-Signed" to self-signed (no quote; no caps)

File: docs-source/mbaas/content/setup/_index.md [hunk 2]
```diff
@@ -64,25 +64,47 @@ To start the installation, take the following steps:
...
+
+### Updating the TLS Certificate
+
+1. Ensure your DNS entry points to the IP Address specified in the `parse_dahsboard_uri` output.
```
Comment (rhondaday) [inline docs-source/mbaas/content/setup/_index.md:94]: Change Address to address

**Review by rhondaday — CHANGES_REQUESTED** [review]

**Review by markxnelson — DISMISSED** [review]
LGTM

File: docs-source/mbaas/content/setup/_index.md [hunk 3]
```diff
@@ -64,25 +64,47 @@ To start the installation, take the following steps:
...
-dashboard_uri = "http://1.2.3.4"
-dashboard_user = "ADMIN"
-kubeconfig_cmd = "oci ce cluster create-kubeconfig --cluster-id ocid1.cluster.oc1.iad.xxx
- --file $HOME/.kube/config --region us-ashburn-1 --token-version 2.0.0 --kube-endpoint PUBLIC_ENDPOINT"
-parse_endpoint = "1.2.3.4/parse"
-```
+
+## TLS
+
+The Oracle Backend for Parse Platform is deployed with a sample self-signed certificate for TLS.  This results in an "Accept Risk" message when accessing the Parse Dashboard and the sample TLS certificate should not be used for production deployments.
```
Comment (rhondaday) [inline docs-source/mbaas/content/setup/_index.md:90]: In the first sentence, change "TLS" to "Transport Layer Security (TLS)".

It looks like there may be an extra space between sentences. There should only be one space between sentences.

File: docs-source/mbaas/content/setup/_index.md [hunk 4]
```diff
@@ -64,25 +64,47 @@ To start the installation, take the following steps:
...
+
+### Updating the TLS Certificate
+
+1. Ensure your DNS entry points to the IP address specified in the `parse_dashboard_uri` output.
```
Comment (rhondaday) [inline docs-source/mbaas/content/setup/_index.md:94]: Change to:

1. Ensure that your Domain Name System (DNS) entry ...

**Review by rhondaday — COMMENTED** [review]
Just a couple of comments.

**Review by rhondaday — APPROVED** [review]
Looks good.

**Pull request merged**